kpi/tests/api/v2/test_api_attachments_delete_viewset.py

```diff
@@ -337,13 +430,11 @@ def test_bulk_delete_attachments_unauthenticated(self):
         }
 
     def test_bulk_delete_not_shared_attachment_as_anotheruser(self):
-        another_user = User.objects.create(
-            username='another_user', password='another_user'
-        )
-        another_user.user_permissions.clear()
-        self.asset.assign_perm(another_user, PERM_VIEW_SUBMISSIONS)
```

Comment:
`anotheruser` already exists. It was confusing to create another one with almost the same name.